packages/services/f-braze-adapter/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-braze-adapter",
   "description": "Fozzie Metadata Service",
-  "version": "4.0.0-beta.1",
+  "version": "4.0.0-beta.x",
```

Comment:
What does the `x` signify?

Reply:
Its because @benwhite-deltas does not know the version it will be as I am making PR's under the beta branch atm, so assume its just holding while in draft

Reply:
@DamianMullins - It signifies that the version number is not finalised yet, as it's a draft PR, and there is another piece of work being worked on that will potentially come in before this. I'll update to be a digit once it's ready.

Reply:
Cool, thanks.